tests/OmniSharp.Roslyn.CSharp.Tests/SignatureHelpFacts.cs

```diff
@@ -224,6 +224,32 @@ public MyTestAttribute(int value1,double value2)
             Assert.Equal("value2", signature.Parameters.ElementAt(1).Name);
             Assert.Equal("double value2", signature.Parameters.ElementAt(1).Label);
         }
+        [Fact]
+        public async Task SignatureHelpforAttributeCtorActiveParamBasedOnComma()
+        {
+            const string source =
+@"using System;
+[MyTest(2,$$)]
+public class TestClass 
+{
+    public static void Main()
+    {
+    }
+}
+public class MyTestAttribute : Attribute 
+{
+    int value1;
```

Comment:
Up to you, but you can remove the fields and their assignment, if you want to make the test code even simpler.